### PR TITLE
Test install on every build

### PR DESCRIPTION
**Pull request by jumaffre**
It is only tested on `Release` builds at the moment.

### CODE REVIEW

**Comment by eddyashton** [comment]
"While you're here": I just noticed we don't include `parameters.build.common.cmake_args` in `cmake_args` for the _Perf_ tests (`matrix.yml`, line 110). This means they're not using ccache, so they build a lot slower. Can you add it in this PR too?

**Comment by achamayou** [comment]
@eddyashton so in principle, if the install now works well enough, we *should* be able to use it for Perf tests rather than rebuild.

**Comment by cimetrics** [comment]
test_install_on_every_build@[5579 aka 20200303.37](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=5579&view=results) vs master ewma over 23 builds from [5233](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=5233&view=results) to [5545](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=5545&view=results)
images

File: .azure-pipelines-templates/ninja.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ steps:
   displayName: Ninja
   workingDirectory: build
   env:
-    CCACHE_DIR: /ccache
+    CCACHE_DIR: /tmp/ccache
```
Comment (achamayou) [inline .azure-pipelines-templates/ninja.yml]: Why this change?

Comment (eddyashton) [reply .azure-pipelines-templates/ninja.yml]: ccache tries to create this directory, but when this isn't running in docker the job [doesn't have permission](https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=5557&view=logs&j=9be4a519-996c-5dea-a60e-59d3ff2364b1&t=7e4c56d2-1b54-52d4-b692-8716a14d9d94&l=37) to create `/ccache`.
